server/src/index.js

```diff
@@ -20,11 +21,14 @@ server.use("/api/locations", LocationsRouter);
 server.use("/api/menus", MenusRouter);
 server.use("/api/menuItems", MenuItemsRouter);
 server.use("/api/favorites", FavoritesRouter);
+server.use("/api/testPush", TestPushRouter);
 
 server.listen(PORT, e => e && console.error(e));
 
 const options = {
     scheduled: true,
     timezone: "America/New_York"
 };
-cron.schedule("0 0 7 * * *", () => sendNotifications(), options);
+
+// Run at 7:00 am every day
+cron.schedule("0 7 * * *", () => sendNotifications(), options);
```

Comment:
This is actually once every hour at the 7 minute mark, I think.

Reply:
I think there are 2 formats - 
`<Minute> <Hour> <Day_of_the_Month> <Month_of_the_Year> <Day_of_the_Week>`
and
`<Minute> <Hour> <Day_of_the_Month> <Month_of_the_Year> <Day_of_the_Week> <Year>`

By this, I think the original would mean a notification at midnight on the 7th of every month. 

Used [this tool](https://crontab.guru/#0_7_*_*_*) to verify

Reply:
You're right! I neglected the fact that the second parameter is optional.